lib/pf/firewallsso/PaloAlto.pm

```diff
@@ -67,6 +67,9 @@ XML
             my $webpage = "https://".$firewall_conf."/api/?type=user-id&action=set&key=".$ConfigFirewallSSO{$firewall_conf}->{'password'};
             my $ua = LWP::UserAgent->new;
             $ua->timeout(5);
+            # to work with perl 5.x (Debian 8 Jessie) and self-signed certificate
+            $ua->ssl_opts( verify_hostname => 0 ,SSL_verify_mode => 0x00);
```

Comment:
Guys.
I believe this should be a configuration value not hardcoded.
What are you thoughts.